pvlib/pvsystem.py

```diff
@@ -129,17 +129,17 @@ class PVSystem:
         Azimuth angle of the module surface.
         North=0, East=90, South=180, West=270.
 
-    albedo : None or float, default None
+    albedo : float, optional
         Ground surface albedo. If ``None``, then ``surface_type`` is used
         to look up a value in ``irradiance.SURFACE_ALBEDOS``.
         If ``surface_type`` is also None then a ground surface albedo
```

Comment:
```suggestion
        If ``surface_type`` is not specified then a ground surface albedo
```

Reply:
Gotta account for "also" in the regexes...